content/500-support/100-help-articles/500-comparing-columns-through-raw-queries.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Comparing columns of same table through raw queries'
+metaTitle: 'Comparing columns of same table through raw queries'
+metaDescription: 'Comparing columns of same table through raw queries'
+---
+
+## Workaround
+
+Comparing columns from a same table is a common scenario which developers have to deal with. Some examples of it could be comparing two numeric values in a same table or comparing two dates in a same table.
+
+
+## Solution
+
+This could be achieved by using [Raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access)
```

Comment:
Since we never want to start a sentence with "This" or "It" or anything that is non-specific, let's reword this to something like: "Comparing values from two columns in the same table can be achieved by using raw queries..."

Reply:
Updated the sentence